dev/validate_bom__test.py

```diff
@@ -1140,8 +1131,9 @@ def init_argument_parser(parser, defaults):
       help='The Jenkins job name to use in tests.')
 
   add_parser_argument(
-      parser, 'test_gate_service_host', defaults, None,
-      help='Gate host to use rather than port-forwarding.')
+      parser, 'test_gate_service_endpoint', defaults, None,
```

Comment:
if this is a url then the name should say url, like "test_gate_service_url".  Then it is clear it is specified as a url.

Reply:
Agreed. Would you be opposed to "test_gate_service_base_url"?

Reply:
SGTM